### DISCUSSION TITLE
Error message when I try to run the downloader

### DISCUSSION THREAD

**Discussion by Lieutenant-Lettuce** [X2CommunityCore/xcom2-launcher#261] [Help]
when I try to launch this to download it I get an error message saying that it failed to create steam_appid.txt and access to the path is denied and I have no idea why or how to fix it.

**Comment by RevZero** [comment]
You probably copied/extracted AML to a folder that is write protected.

See https://github.com/X2CommunityCore/xcom2-launcher/wiki/Installation

**Reply by Lieutenant-Lettuce** [reply]
I made the folder on my desktop.
